src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -4374,8 +4374,7 @@ operand immI8()
 // 8 bit signed value (simm8), or #simm8 LSL 8.
 operand immI8_shift8()
 %{
-  predicate((n->get_int() <= 127 && n->get_int() >= -128) ||
-            (n->get_int() <= 32512 && n->get_int() >= -32768 && (n->get_int() & 0xff) == 0));
+  predicate(Assembler::operand_valid_for_sve_dup_immediate((int64_t)n->get_int()));
```

Comment:
`Assembler::operand_valid_for_sve_dup_immediate` sounds odd as the predicate for a generically sounding `immI8_shift8`. These operands are only used in `replicate` rules, though. So we might be taking precedent from immIAddSubV` rule:

```
// 32 bit integer valid for vector add sub immediate
operand immIAddSubV()
%{
  predicate(Assembler::operand_valid_for_sve_add_sub_immediate((int64_t)n->get_int()));
  match(ConI);

  op_cost(0);
  format %{ %}
  interface(CONST_INTER);
%}
```

I.e. rename these operands to `immIDupV`, `immLDupV`, `immHDupV` and adjust the comments to match?

Reply:
Done